packages/driver/cypress/e2e/commands/actions/click.cy.js

```diff
@@ -546,6 +546,27 @@ describe('src/cy/commands/actions/click', () => {
       cy.getAll('span2', 'focus click mousedown').each(shouldNotBeCalled)
     })
 
+    // https://github.com/cypress-io/cypress/issues/28788
+    it('no click when element is disabled', () => {
+      const btn = cy.$$('button:first')
+      const span = $('<span>foooo</span>')
+
+      attachFocusListeners({ btn, span })
+      attachMouseClickListeners({ btn, span })
+      attachMouseHoverListeners({ btn, span })
+
+      btn.html('')
+      btn.attr('disabled', true)
+      btn.append(span)
+
+      cy.get('button:first span').click()
+
+      cy.getAll('btn', 'mouseenter mousedown mouseup').each(shouldBeCalled)
```

Comment:
Looks like this test is failing in CI on this line @TheoAnastasiadis https://app.circleci.com/pipelines/github/cypress-io/cypress/59642/workflows/13231e3d-4919-4cf4-87e0-59f684beb5d4/jobs/2482293/tests#failed-test-0. If I get some time I should be able to see what is going on